pytorch_lightning/core/mixins/device_dtype_mixin.py

```diff
@@ -177,7 +179,9 @@ def __update_properties(
         self, device: Optional[torch.device] = None, dtype: Optional[Union[str, torch.dtype]] = None
     ) -> None:
         def apply_fn(module: Union["DeviceDtypeModuleMixin", Module]) -> None:
-            if not isinstance(module, DeviceDtypeModuleMixin):
+            # TODO: Find why `isinstance(module, DeviceDtypeModuleMixin)` doesn't
+            # work when using `init_meta_device`.
```

Comment:
Isn't it worth trying to resolve this TODO here? might be because we have a bug

Reply:
The training seems to work fine without it. I have no idea why it doesn't work.